packages/mobile/src/web3/contracts.ts

```diff
@@ -1,14 +1,29 @@
+import { addLocalAccount as web3utilsAddLocalAccount } from '@celo/walletkit'
 import { Platform } from 'react-native'
 import { DocumentDirectoryPath } from 'react-native-fs'
 import * as net from 'react-native-tcp'
+import { GethSyncMode } from 'src/geth/consts'
+import config from 'src/geth/network-config'
 import Logger from 'src/utils/Logger'
 import { DEFAULT_TESTNET, Testnets } from 'src/web3/testnets'
 import Web3 from 'web3'
+import { Provider } from 'web3/providers'
 
 // Logging tag
 const tag = 'web3/contracts'
 
-const getWeb3IpcProvider = (testnet: Testnets) => {
+export const web3: Web3 = getWeb3()
+
+export function isZeroSyncMode(): boolean {
+  return config[DEFAULT_TESTNET].syncMode === GethSyncMode.ZeroSync
+}
+
+export function getInfuraUrl(): string {
+  return `https://${DEFAULT_TESTNET}-infura.celo-testnet.org/`
```

Comment:
Suggest moving the base URL to config.ts

Reply:
Created `DEFAULT_INFURA_URL` in `src/config.ts`